### ISSUE TITLE
F3 console key opens search under firefox

### ISSUE THREAD

**Issue by soronpo** [scalacenter/scastie#924]
### Describe the bug

F3 console key opens search under firefox

### To Reproduce

(tested under windows)
1. Go to any scastie under firefox
2. Click on F3 key
3. Firefox opens search instead of toggling the console window

### Expected behavior

Maybe its possible to force the trapping of F3 under firefox, otherwise I suggest one of the following:
* change to a key that is supported under all browsers.
* change a key just for firefox and display that key accordingly
* if no key trapping is possible under firefox, then remove the key under firefox to avoid user confusion.

### Version

Scastie website

### What browsers are you seeing the problem on?

Firefox

**Comment by warcholjakub** [comment]
Thank you for reporting this!

Based on my testing, even if we change the shortcut to (for example) F4, the behavior does not really improve, the shortcut will still only work when the code editor is focused. If the editor is not focused, the browser’s default action (or no action) will take precedence. Also, for me, when the editor is focused in Firefox, F3 works fine, so I was not able to reproduce the bug - perhaps there are differences depending on the browser version.

**Comment by soronpo** [comment]
Yes, indeed the code editor needs to be in focus. Is it not possible to create key trapping within the window (tab) instead?

**Comment by warcholjakub** [comment]
It should be possible to implement global key trapping within the browser tab. However, we need to make sure to use a shortcut that does not conflict with any browser defaults. I’ll look into this.

**Comment by soronpo** [comment]
> Okay, from what I tested, it is indeed possible, but another issue appears: when we use global trapping, the keybind stops working when the editor is focused.

What if you enable both?

**Comment by warcholjakub** [comment]
I tested it with both enabled. But maybe it's possible to make it work. I'll look into that.
